### PR TITLE
PR for issue #5118 "rename beaker object to beakerx"

### PR DESCRIPTION
**Pull request by konstantinTarletski**

### CODE REVIEW

File: beakerx/beakerx/bkr2ipynb.py [hunk 1]
```diff
@@ -11,11 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from statsmodels.datasets import nile
```
Comment (scottdraves) [inline beakerx/beakerx/bkr2ipynb.py]: what is this?

Comment (konstantinTarletski) [reply beakerx/beakerx/bkr2ipynb.py]: I do not know how I add it , sorry

**Comment by scottdraves** [comment]
trying to start jupyter results in
```
[C 11:34:39.427 NotebookApp] The 'kernel_spec_class' trait of <jupyter_client.kernelspec.KernelSpecManager object at 0x108b0e5c0> instance must b\
e a type, but 'beaker.kernel_spec.BeakerXKernelSpec' could not be imported
```

**Comment by konstantinTarletski** [comment]
Today I did clean install, fix one wrong name bug in "install.py"
But I do not get this error on my Windows PC.
For me it works, no any errors.

**Comment by scottdraves** [comment]
remaining places where "beaker" -> "beakerx":

1) In autotranslation_python.ipynb `beaker = beakerx.runtime.Beaker()` should be `beakerx = beakerx.runtime.BeakerX()`

2) in autotranslation_groovy_demo.ipynb
```
beaker.displayHTML(this, '<div id="fdg"></div>');
var graph = beaker.graph
```
should be
```
beakerx.displayHTML(this, '<div id="fdg"></div>');
var graph = beakerx.graph
```

3) move the java package `com.twosigma.beaker` to `com.twosigma.beakerx`

4) in plotActions.ipynb when I run 3rd cell i get this console error:
```
Exception in thread "Thread-0" groovy.lang.MissingPropertyException: No such property: beaker for class: script1496841851688163566422
```

**Pull request merged**